mestopy/mestopy.py

```diff
@@ -1 +1,242 @@
-"""Main module."""
+import numpy as np
+from pyfar import Signal
+
+
+# Class to generate ref-Objects, that can bei part of the MeasurementChain
+class DeviceObj(object):
+    """Class for device in MeasurementChain.
+
+    This class holds methods and properties of a device in the
+    'MeasurementChain' class.
+    """
+
+    def __init__(self, data, sens=1, name=''):
```

Comment:
I would suggest the following
- `name` should be the first attribute without a default. This way the user has to give a name, which makes the thing more robust
- `data=None` could be a good default. If data is none, we assume a frequency dependent sensitivity
- add `unit=None`. The sole purpose of this would be to display the unit in __repr__ and have a property